src/sql/src/statements/create.rs

```diff
@@ -310,6 +312,9 @@ impl Display for CreateView {
             write!(f, "IF NOT EXISTS ")?;
         }
         write!(f, "{} ", &self.name)?;
+        if !self.columns.is_empty() {
+            write!(f, "({}) ", format_list_comma!(self.columns))?;
+        }
         write!(f, "AS {}", &self.query)
```

Comment:
Is this `Display` impl tested?

Reply:
Tested in `create_parser.rs`, just like:

```
assert_eq!(sql, result[0].to_string());
```

Assert the `to_string` result is the same as the raw SQL.